HOWTO.md

```diff
@@ -446,10 +446,10 @@ service_info:
     sshkeys:
     - "testkey"
   files:
-  - path: /device/etc/hosts
+  - path: /var/lib/fdo/etc/hosts
     permissions: 644
     source_path: /server/local/etc/hosts
-  - path: /device/etc/resolv.conf
+  - path: /var/lib/fdo/etc/resolv.conf
```

Comment:
I think we would want anything the service-info API sends to be in a sub directory under /var/lib/fdo and not in the root directory. Ultimately we're going to end up with DBs and potentially a raft of other things and we likely don't want that pouted with a bunch of random files.

Reply:
So likely something like /var/lib/fdo/service-info-api/files/

Reply:
there's nothing enforcing that but I can definitely update the examples

Reply:
right, I think we should be setting a decent example here, and possibly in the next update actually doing more setup around directories, but that's a different RFE.